config/default.json

```diff
@@ -1,21 +1,10 @@
 {
   "PORT": 3000,
   "HOSTNAME": "http://ml-api-adapter",
-  "DFSP_URLS": {
-    "dfsp1": {
-      "transfers": {
-        "post": "http://localhost:1080/dfsp1/transfers",
-        "put": "http://localhost:1080/dfsp1/transfers/{{transferId}}",
-        "error": "http://localhost:1080/dfsp1/transfers/{{transferId}}/error"
-      }
-    },
-    "dfsp2": {
-      "transfers": {
-        "post": "http://localhost:1080/dfsp2/transfers",
-        "put": "http://localhost:1080/dfsp2/transfers/{{transferId}}",
-        "error": "http://localhost:1080/dfsp2/transfers/{{transferId}}/error"
-      }
-    }
+  "ENDPOINT_CACHE": {
```

Comment:
Please clarify ENDPOINT_CACHE?

I do not believe the cache and the URI should be under the same configuration.